core/save-xml.c

```diff
@@ -119,12 +119,60 @@ static void save_salinity(struct membuffer *b, struct divecomputer *dc)
 	put_string(b, " />\n");
 }
 
-static void save_overview(struct membuffer *b, struct dive *dive)
+static void xxxout(char *c)
 {
-	show_utf8(b, dive->divemaster, "  <divemaster>", "</divemaster>\n", 0);
-	show_utf8(b, dive->buddy, "  <buddy>", "</buddy>\n", 0);
-	show_utf8(b, dive->notes, "  <notes>", "</notes>\n", 0);
-	show_utf8(b, dive->suit, "  <suit>", "</suit>\n", 0);
+	while(*c) {
+		switch (*c) {
+		case 'A'...'Z':
+			*c = 'X';
+			break;
+		case 'a'...'z':
+			*c = 'x';
+			break;
+		default:
+			;
+		}
+		++c;
+	}
+}
+
+static void save_overview(struct membuffer *b, struct dive *dive, bool annonymize)
+{
+	char *xxxs;
```

Comment:
again, XXX is unfortunately a standing expression I'd prefer we'd stay away from.